understanding/label-in-name.html

```diff
@@ -35,30 +35,30 @@ <h1>達成基準 2.5.3: 名前 (name) のラベルを理解する</h1>
       <main>
          <section id="intent">
             <h2>意図</h2>
-            <p>The intent of this Success Criterion (SC) is to help ensure that people with disabilities who rely on visual labels can also use those labels programmatically. Controls are often labeled with visible text. Controls also have a programmatic label, known as its <a href="https://www.w3.org/TR/accname-aam-1.1/">Accessible Name</a>. Users have a much better experience if the visible text labels of controls match their accessible names.</p>
-            <p>Speech input users can navigate by speaking the visible text labels of menus, links and buttons that appear on the screen. It&#39;s confusing to speech input users when they say a visible text label they see, but the speech input does not work because the accessible name that is enabled as a speech input command does not match the visible label.</p>
-            <p>In addition, when the accessible names of controls are visually hidden, speech users may accidentally activate these controls when they happen to speak words that map to these hidden accessible names.</p>
-            <p>Text-to-speech users will also have a better experience if the text they hear matches the text they see on the screen.</p>
-            <p>Sometimes authors want to add additional text to provide more context to a link or other control to make it more descriptive for users of assistive technologies. In this case, it is a recommended practice to add that text <strong>after</strong> the visible label so the accessible name starts with the visible label. In the case of the search button, the accessible name could be &quot;search this site&quot;, in the case of the home link, the text could be &quot;home - go to start page&quot;. In both cases, the hidden portion of the label may be supplied by different means, for example, as text accessibly hidden via CSS or by using the aria-label or the aria-labelledby attribute.</p>
-            <p>This issue is even more important for speech input and text-to-speech users who also have cognitive challenges. It’s an extra cognitive load for a speech input user to remember and say a speech command that is different from the visible label they see on a control. It’s also an extra cognitive load for a text-to-speech user to absorb and understand speech output that does not match the visible label.</p>
+            <p>この達成基準の意図は、視覚的なラベルに依存する障害者が、プログラムでもこれらのラベルを使用できるようにすることである。多くの場合、コントロールには視認できるテキストのラベルが付けられる。コントロールには、加えて、アクセシブルな名前 (<a href="https://www.w3.org/TR/accname-aam-1.1/">Accessible Name</a>) と呼ばれるプログラムによって利用可能なラベルもある。コントロール上の視覚的なテキストラベルが、アクセシブルな名前と一致している場合、利用者ははるかによりよい体験が得られる。</p>
+            <p>音声入力を行う利用者は、画面に表示されるメニュー、リンク、ボタンのテキストラベルを読み上げることで、ナビゲートすることができる。音声入力を行う利用者が、目にした視覚的なテキストラベルを読み上げながらも、音声入力コマンドとして有効なアクセシブルな名前がその視覚的なラベルと一致していないことで音声入力が正常に機能しない場合、利用者は混乱してしまう。</p>
```

Comment:
「**because** the accessible name that is enabled as a speech input command does not match the visible label」は、「音声入力コマンドとして有効なアクセシブルな名前がその視覚的なラベルと一致していない**ことで**」と訳しています。